Include/refcount.h

```diff
@@ -354,7 +374,13 @@ static inline void Py_DECREF(PyObject *op)
 #elif defined(Py_REF_DEBUG)
 static inline void Py_DECREF(const char *filename, int lineno, PyObject *op)
 {
+#if SIZEOF_VOID_P > 4
+    /* If an object has been freed, it will have a negative full refcnt
+     * If it has not it been freed, will have a very large refcnt */
+    if (op->ob_refcnt_full <= 0 || op->ob_refcnt > (UINT32_MAX - (1<<20))) {
```

Comment:
Apparently `UINT32_MAX` is not defined in all builds, so this [breaks a buildbot](https://buildbot.python.org/#/builders/1223/builds/4778).

Reply:
We use `UINT32_MAX` all over the place, so it must be defined. Maybe we're missing a `#include` somewhere.

Reply:
Looks like it's a C++ build that failing, which might explain it

Reply:
https://github.com/python/cpython/pull/127863